openspace.cfg

```diff
@@ -228,10 +228,10 @@ ShutdownCountdown = 3
 ScreenshotUseDate = true
 BypassLauncher = false
 
-PreferedLayerServer = "All"
--- PreferedLayerServer = "NewYork"
--- PreferedLayerServer = "Sweden"
--- PreferedLayerServer = "Utah"
+-- LayerServer = "All"
```

Comment:
I think we should keep the default as "All" until we automatically detect the location

Reply:
Yea ofcourse, ops ^^'